modules/cco-ccoctl-deleting-sts-resources.adoc

```diff
@@ -4,50 +4,82 @@
 // * installing/installing_gcp/uninstalling-cluster-gcp.adoc
 
 ifeval::["{context}" == "uninstall-cluster-aws"]
+:cp-first: Amazon Web Services
+:cp: AWS
+:cp-name: aws
 :aws-sts:
 endif::[]
 ifeval::["{context}" == "uninstalling-cluster-gcp"]
-:google-cloud-platform:
+:cp-first: Google Cloud Platform
+:cp: GCP
+:cp-name: gcp
+:gcp-workload-id:
+endif::[]
+ifeval::["{context}" == "uninstall-cluster-azure"]
+:cp-first: Microsoft Azure
+:cp: Azure
+:cp-name: azure
+:azure-workload-id:
 endif::[]
 
 :_content-type: PROCEDURE
 [id="cco-ccoctl-deleting-sts-resources_{context}"]
-ifdef::aws-sts[]
-= Deleting AWS resources with the Cloud Credential Operator utility
-
-To clean up resources after uninstalling an {product-title} cluster with the Cloud Credential Operator (CCO) in manual mode with STS, you can use the CCO utility (`ccoctl`) to remove the AWS resources that `ccoctl` created during installation.
-endif::aws-sts[]
-
-ifdef::google-cloud-platform[]
-= Deleting GCP resources with the Cloud Credential Operator utility
+= Deleting {cp-first} resources with the Cloud Credential Operator utility
 
-To clean up resources after uninstalling an {product-title} cluster with the Cloud Credential Operator (CCO) in manual mode with GCP Workload Identity, you can use the CCO utility (`ccoctl`) to remove the GCP resources that `ccoctl` created during installation.
-endif::google-cloud-platform[]
+After uninstalling an {product-title} cluster that uses short-term credentials managed outside the cluster, you can use the CCO utility (`ccoctl`) to remove the {cp-first} ({cp}) resources that `ccoctl` created during installation.
 
 .Prerequisites
 
 * Extract and prepare the `ccoctl` binary.
-ifdef::aws-sts[]
-* Install an {product-title} cluster with the CCO in manual mode with STS.
-endif::aws-sts[]
-ifdef::google-cloud-platform[]
-* Install an {product-title} cluster with the CCO in manual mode with GCP Workload Identity.
-endif::google-cloud-platform[]
+* Uninstall a {cp} cluster that uses short-term credentials.
 
 .Procedure
-
-ifdef::aws-sts[]
-* Delete the AWS resources that `ccoctl` created:
+//GCP has extra prep steps
+ifdef::gcp-workload-id[]
+. Set a `$RELEASE_IMAGE` variable with the release image from your installation file by running the following command:
 +
 [source,terminal]
 ----
-$ ccoctl aws delete \
-  --name=<name> \ <1>
-  --region=<aws_region> <2>
+$ RELEASE_IMAGE=$(./openshift-install version | awk '/release image/ {print $3}')
+----
+
+. Extract the list of `CredentialsRequest` custom resources (CRs) from the {product-title} release image by running the following command:
++
+[source,terminal,subs="attributes+"]
+----
+$ oc adm release extract \
+  --credentials-requests \
+  --included \// <1>
+  --install-config=<installation_directory>/auth/kubeconfig <2>
```

Comment:
Took a guess based on existing docs about `kubeconfig` and Jianping's comment that we need to point to that instead of `install-config.yaml`. @wking or @jianping-shu please correct this since I probably guessed wrong :slightly_smiling_face:

Reply:
You are right!

Reply:
I think we can drop this line.

* Kubeconfigs are not install-configs, so `--install-config` here will fail with "hey, this file content is not an install config".
* We have docs for lots and lots of `oc ...` commands that just assume the user has figured out how to access a cluster via a kubeconfig.  For example, [this existing docs][1] have:

> Prerequisites
> * You have access to an OpenShift Container Platform account with cluster administrator permissions.
> * You have installed the OpenShift CLI (oc).

and then when they tell you to use `oc`, they say:

> To determine the mode that the CCO is configured to use, enter the following command:
> ```console
> $ oc get cloudcredentials cluster -o=jsonpath={.spec.credentialsMode}
> ```

without talking about kubeconfigs or [profiles][2] or any of that stuff.

[1]: https://docs.openshift.com/container-platform/4.13/updating/preparing-manual-creds-update.html#cco-determine-mode-cli_preparing-manual-creds-update
[2]: https://docs.openshift.com/container-platform/4.13/cli_reference/openshift_cli/managing-cli-profiles.html